lib/fortnox/api/validators/constant.rb

```diff
@@ -0,0 +1,20 @@
+module Fortnox
+  module API
+    module Validator
+      module Constant
+
+        def validate( _instance )
```

Comment:
Just `_` works for me. I know either is ok according to different coding standards, but you really don't care what it is you get in here ...